core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -164,6 +165,8 @@ var CollectionEditorPage = function() {
     saveChangesButton.click();
     waitFor.invisibilityOf(
       saveChangesButton, 'Save Changes modal takes too long to close');
+    waitFor.invisibilityOf(
+      saveInProgressLabel, 'Collection is taking too long to save.');
```

Comment:
Wanted to check why we are expecting to have "saveInProgressLabel" after "saveChangesButton", right? (Not sure about the expected behaviour of these buttons and workflow because of their names.)

Reply:
The saveInProgressLabel actually refers to a temporary label we show when trying to save/publish the collection. Hm, thinking about it now, I guess we could rename it to "saveOrPublishInProgressLabel".